### ISSUE TITLE
ResultEd: Inconsistant default filter names

### ISSUE THREAD

**Issue by c17alepe** [HGustavs/LenaSYS#7747]
![image](https://user-images.githubusercontent.com/38323901/62052660-346b9080-b216-11e9-9796-71b10f618c41.png)

The naming for not using the filters are inconsistant

**Comment by b18alegu** [comment]
Note: @a18ismca and @a17pioja, I'll move the pull request to G2-2020-W14 branch from master.

**Comment by b18alegu** [comment]
This issue is solved and merged into the G2-2020-W14 branch
Issue will be closed

**Comment by a18ismca** [comment]
This issue needs to be merged into this week's branch as this issue was not "fixed" in the latest branching, even though the changes were made in week 14. The issue was fixed for week 15 but as everyone was merging so the issues seems not to be fixed for week 16 (this week). Although I fixed the minor issue in resultet.php into G2-2020-W16 today so maybe this issue could be closed after all. Still linked to same pull request.
